### ISSUE TITLE
Welcome pages are empty and the help windows do not open

### ISSUE THREAD

**Issue by erikmd** [jcryptool/core#105]
Hello,

I am experiencing the following display issue with JCrypTool's latest weekly build:
At startup or when I do Help > Welcome, I obtain a blank page.
In addition, when I do Help  > Help Contents, the internal help window doesn't load (a help page opens in my external browser instead).

My system is Ubuntu 14.04.2 LTS x86_64 with GTK 3.10.8 and OpenJDK 8u45.

Note that this issue is unrelated to #52 but I suspect that it has something to do with GTK...
For example (borrowing an idea from [Eclipse-bug-421127](https://bugs.eclipse.org/bugs/show_bug.cgi?id=421127)):
running the command `$ SWT_GTK3=1 ./JCrypTool` seems to workaround the problems above, but it makes all balloons/tooltips unreadable: both foreground and background of these tooltips are black...

Would you have a fix or a better workaround ?

Best regards.

**Comment by dschadow** [comment]
Hi,

I have given JCT a try on Linux MINT 17.2 and everything seems to work. My GTK version and OpenJDK are identical with yours. 

Is the rest working? E.g. do the views (like Algorithms or any Visualization plug-in) contain data? 

Regards, Dominik

**Comment by erikmd** [comment]
Hello Dominik,

Thanks for your reply; Yes I confirm that the rest is working.

Regards.

**Comment by dschadow** [comment]
Hi Erik,

At least something is working on Ubuntu. Can you please send me your log file? This file is located at /Users/[USERNAME]/Documents/.jcryptool/.metadata and is named .log (directory and file are hidden). It would make the analysis much easier if you delete this file (or the complete .jcryptool directory in Documents), start JCT and click the not working help actions. Hopefully there is an exception available in the log file now... Either attach this file here or mail it to dominikschadow@gmail.com. 

Regards, Dominik

**Comment by erikmd** [comment]
Hi Dominik,

Sorry for the delay.
I removed `~/Documents/.jcryptool`, restarted JCT (the weekly build as of 2015-07-30), closed the blank Welcome page so obtained (cf. screenshot below), and selected Help > Help Contents.
![jct_first_startup](https://cloud.githubusercontent.com/assets/10367254/9092152/95f56778-3ba4-11e5-8e49-af5b8356d8c5.png)
Unfortunately the `~/Documents/.jcryptool/.metadata/.log` file below does not seem very helpful (the only exception is related to connecting to www.cryptool.org, as I was offline at that time).
As I said before this display issue seems to be related to SWT and GTK (I have the packages libgtk2.0-0 version 2.24.23-0ubuntu1.2 and libgtk-3-0 version 3.10.8-0ubuntu1.6): setting the environment variable `SWT_GTK3=1` solves this issue... but breaks various other aspects of the interface, including tooltips, so it is not a working solution for me.
Anyway, please let me know if I have to do more tests.

Best regards.
```
!SESSION 2015-08-05 18:46:22.048 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_45-internal
java.vendor=Oracle Corporation
BootLoader constants: OS=linux, ARCH=x86_64, WS=gtk, NL=en
Command-line arguments:  -os linux -ws gtk -arch x86_64 -data @user.home/Documents/.jcryptool

!ENTRY org.eclipse.equinox.p2.transport.ecf 2 0 2015-08-05 18:46:27.275
!MESSAGE Connection to http://www.cryptool.org/jct/update/p2.index failed on www.cryptool.org: unknown error. Retry attempt 0 started
!STACK 0
java.net.UnknownHostException: www.cryptool.org: unknown error
	at java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.net.InetAddress$2.lookupAllHostAddr(InetAddress.java:907)
	at java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1302)
	at java.net.InetAddress.getAllByName0(InetAddress.java:1255)
	at java.net.InetAddress.getAllByName(InetAddress.java:1171)
	at java.net.InetAddress.getAllByName(InetAddress.java:1105)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:259)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:144)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:131)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:611)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:446)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer.performConnect(HttpClientRetrieveFileTransfer.java:1077)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer.access$0(HttpClientRetrieveFileTransfer.java:1068)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer$1.performFileTransfer(HttpClientRetrieveFileTransfer.java:1064)
	at org.eclipse.ecf.filetransfer.FileTransferJob.run(FileTransferJob.java:73)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.equinox.p2.transport.ecf 4 1006 2015-08-05 18:46:27.545
!MESSAGE Unknown Host: http://www.cryptool.org/jct/update/content.xml
!STACK 0
java.net.UnknownHostException: www.cryptool.org
	at java.net.InetAddress.getAllByName0(InetAddress.java:1259)
	at java.net.InetAddress.getAllByName(InetAddress.java:1171)
	at java.net.InetAddress.getAllByName(InetAddress.java:1105)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:259)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.ManagedClientConnectionImpl.open(ManagedClientConnectionImpl.java:304)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:611)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:446)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientFileSystemBrowser.runRequest(HttpClientFileSystemBrowser.java:259)
	at org.eclipse.ecf.provider.filetransfer.browse.AbstractFileSystemBrowser$DirectoryJob.run(AbstractFileSystemBrowser.java:69)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
```

**Comment by dschadow** [comment]
Hi Erik,

Thanks for all your testing! This issue is fixed with #106 (update to Eclipse Mars) which will be available with the next weekly build on Saturday. I've installed Ubuntu 14.04.3 with Java 8 and the Welcome Page content is available as well as the help system.

Dominik

**Comment by erikmd** [comment]
Hello Dominik,

Thanks for the bugfix!

Best regards.
